src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList-Search.cshtml

```diff
@@ -1,4 +1,5 @@
 @model ContentOptionsViewModel
+@removeTagHelper Microsoft.AspNetCore.Mvc.TagHelpers.InputTagHelper, Microsoft.AspNetCore.Mvc.TagHelpers
```

Comment:
Why do you need to do this?

Reply:
Because when you do asp-for="Model.Something" on an input there is a bug where it will append twice the type="search" attribute. This is because of the InputTagHelper in ASP.NET MVC.

See : https://stackoverflow.com/questions/42544961/asp-net-core-custom-input-tag-helper-rendering-duplicate-checkboxes which is somehow similar to that.

Reply:
maybe I am being dumb.

If you've removed the `InputTagHelper` which TagHelper is rendering the 

`<input asp-for="DisplayText"` in line 14?